src/types/common/spr-slot.ts

```diff
@@ -0,0 +1,14 @@
+// * Extracted from https://github.com/MrNbaYoh/libstreetpass/blob/master/include/cec/send_mode.hpp
+export enum SendMode {
+	Exchange,
+	RecvOnly,
+	SendOnly,
+	SendRecv,
```

Comment:
In our JS repos we prefer no trailing commas

```suggestion
	SendRecv
```